test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -1220,6 +1250,64 @@ static uint32_t send_pkts(odp_tm_queue_t tm_queue, uint32_t num_pkts)
 	return pkts_sent;
 }
 
+static uint32_t send_pkts_lso(odp_tm_queue_t tm_queue, uint32_t num_pkts,
+			      odp_lso_protocol_t lso_proto, uint32_t max_len)
+{
+	odp_packet_lso_opt_t lso_opt;
+	odp_lso_profile_t profile;
+	xmt_pkt_desc_t *xmt_pkt_desc;
+	odp_packet_t pkt;
+	uint32_t offset, pkts_sent;
+	int64_t rc, i;
+
+	pkt = xmt_pkts[0];
+
+	if (lso_proto == ODP_LSO_PROTO_IPV4) {
+		profile = lso_ipv4_profile;
+		offset  = odp_packet_l4_offset(pkt);
+	} else {
+		ODPH_ERR("Bad LSO protocol\n");
+		return 0;
+	}
+
+	lso_opt.lso_profile     = profile;
+	lso_opt.payload_offset  = offset;
+	lso_opt.max_payload_len = max_len;
+
+	pkts_sent = 0;
+
+	for (i = 0; i < num_pkts; i++) {
+		pkt = xmt_pkts[i];
+		xmt_pkt_desc = &xmt_pkt_descs[i];
```

Comment:
The indexing here could start from num_pkts_sent instead of zero. It would make this function similar to the other send fumctions in that it can be called more than once after init_xmt_pkts().

Reply:
Fixed in v7